x-pack/plugins/ingest_pipelines/public/application/services/breadcrumbs.ts

```diff
@@ -28,7 +27,7 @@ export class BreadcrumbService {
     create: [
       {
         text: homeBreadcrumbText,
-        href: `#${BASE_PATH}`,
+        href: `/`,
```

Comment:
As noted previously, I think these need an onClick event to nav without page load.

Reply:
I just tested this locally and they actually seem to load OK (no page reload).